cmd/websocktunnel/main.go

```diff
@@ -28,6 +28,7 @@ Environment:
  TASKCLUSTER_PROXY_SECRET_A                  JWT secret
  TASKCLUSTER_PROXY_SECRET_B                  alternate JWT secret
  SYSLOG_ADDR                                 address to which to send syslog output
+ AUDIENCE(optional;default to "wst-servers") JWT 'audience' claim 
```

Comment:
I'd rather not have a default. It's better to be explicit than spend time debugging only to finally see this note and say "oh, *that's* the value I should be using for audience!"

Reply:
Sure, will remove the defaults